test/integration/content-explorer/ContentExplorer.int.test.js

```diff
@@ -102,6 +102,19 @@ describe('ContentExplorer', () => {
     beforeEach(() => {
         cy.server();
         cy.route('GET', '**/folders/*', 'fixture:content-explorer/root-folder.json');
+        cy.route('GET', '**/files/*', {
```

Comment:
The API call for thumbnails was failing, since this Cypress test is using a fixture (line above).  If the API call for thumbnails fails, an error screen appears in the BCE saying a network error occurred.  Stubbing this call seemed to be the best solution.